src/index.js

```diff
@@ -79,11 +80,31 @@ export const getCurrentNodeVersion = () => {
   return parseFloat(process.versions.node);
 };
 
+export const electronVersionToChromeVersion = (semverVer) => {
+  const m = semverVer.match(/^(\d+\.\d+)/);
+  if (!m) {
+    throw new Error("Electron version must be a semver version");
+  }
+
+  let result = electronVersionToChromeVersion[m[1]];
+  if (!result) {
+    throw new Error(`Electron version ${m[1]} is either too old or too new`);
+  }
+
+  return result;
+};
+
 export const getTargets = (targetOpts = {}) => {
   if (targetOpts.node === true || targetOpts.node === "current") {
     targetOpts.node = getCurrentNodeVersion();
   }
 
+  // Rewrite Electron versions to their Chrome equivalents
+  if (targetOpts.electron) {
+    targetOpts.chrome = electronVersionToChromeVersion(targetOpts.electron);
+    delete targetOpts.electron;
```

Comment:
Do you think it's better to delete the option? I guess it makes sense still given it's using the chrome version anyway